### PR TITLE
Check for distance from QT console on interaction

### LINKED ISSUE #228: Quantum telescope can be used from any distance
Submitter: rsg250
Quantum telescope can be used from any distance, even when your not a borg

**To Reproduce**
1. go to QT console and activate
2. walk away
3. continue to ping for roids and such, and locate them
4. see located roids in magnet console
5. See error

**Expected behavior**
you shouldnt be able to do this. humans arent borgs

Reported on: Goonstation 2020-04-13 09:47:36

### PR DESCRIPTION
**Pull request by glassofmilk**
[FIX]

## About the PR 
This PR fixes #228 - Quantum telescope can be used from any distance
The ping animation on the map will still play, since there's not an easy way to check for distance from the javascript, but actual interaction will be prevented.

## Why's this needed? 
Bugfix

### CODE REVIEW

**Comment by Kyle2143** [comment]
This works and is good. But I'm a little surprised that it's necessary. I thought that the QTT implemented a chui window datum, which should implicitly handle user accessibility.

File: code/WorkInProgress/telescope/quantumTelescope.dm [hunk 1]
```diff
@@ -79,6 +79,7 @@ TODO: Enforce ping rate limit here as well in case someone futzes with the javas
 		return
 
 	Topic(href, href_list)
+		if (get_dist(src, usr) > 1) return
```
Comment (flrsh) [inline code/WorkInProgress/telescope/quantumTelescope.dm:82]:
```suggestion
		if (get_dist(src, usr) > 1) 
		  using = null
		  return
```
Set `using` to null so someone else can use it.

Comment (glassofmilk) [reply code/WorkInProgress/telescope/quantumTelescope.dm:82]: With this, if you walk away from the console, poke the map, then walk back within range of the console, you can no longer interact with it until you click the console again, which isn't the way other machines work and feels weird to me. How about a call to bootUser() instead and just forcing the window to close?

Comment (flrsh) [reply code/WorkInProgress/telescope/quantumTelescope.dm]: Good point, that would feel weird. I think you're right, `bootUser()` is more understandable.

**Comment by flrsh** [comment]
Oh wait it should just call `bootUser()` without the distance check. Whoops, will fix.

**Pull request merged**